### PR TITLE
Homework done

### PR DESCRIPTION
**Pull request by DzenAsad**
Please check

### CODE REVIEW

File: Lesson2/src/Homework.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        System.out.println(calculateHypotenuse());
+    }
+
+    /**
+     * 1. Method should return sum of "a" and "b"
+     * 2. (optional) if sum of a and b is more then int can store -- return -1;
+     **/
+    public static int sum(int a, int b) {
+        long t = (long)a+(long)b;
+        if ((long)(a+b) != t) return -1;
```
Comment (GlebLitvinov) [inline Lesson2/src/Homework.java]: when you write (long)(a+b) what happens on runtime - 
first, it sums a and b (as integers), then it casts the result to long. So if a = Integer.MAX_VALUE and b = Integer.MAX_VALUE, then
(long)(a+b) = -2 ,which is equal to just a + b

So you don't really need this cast to long here. Please let me know if you understood my comment =)

File: Lesson2/src/Homework.java [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+     * b = 5
+     * <p>
+     * Method should return 5
+     * Example2:
+     * a = 10,
+     * b = 10
+     * <p>
+     * Method should return 10
+     */
+    public static int max(int a, int b) {
```
Comment (GlebLitvinov) [inline Lesson2/src/Homework.java:38]: good job

File: Lesson2/src/Homework.java [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    }
+
+    /**
+     * Method should return avg. value of given array (sum of all elements of array divided by array length).
+     * <p>
+     * Example:
+     * array = {1,2,3,4,5}
+     * Method should return 3
+     */
+    public static double average(int[] array) {
```
Comment (GlebLitvinov) [inline Lesson2/src/Homework.java:49]: the solution works, but pls do ctrl + alt  + l to fix code formatting

File: Lesson2/src/Homework.java [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+     * 4
+     * Method should return 5
+     * <p>
+     * Example2:
+     * Console input:
+     * 12
+     * 16
+     * Method should return 20
+     */
+    public static double calculateHypotenuse() {
```
Comment (GlebLitvinov) [inline Lesson2/src/Homework.java:83]: good

**Review by GlebLitvinov — CHANGES_REQUESTED** [review]
clean up code by pressing ctrl + alt + l and do a minor fix for the first task

File: Lesson2/src/Homework.java (hunk 1 above)
Comment (DzenAsad) [reply Lesson2/src/Homework.java]: ok i understand, this is redundant cast, delted.

**Review by GlebLitvinov — APPROVED** [review]

**Pull request merged**